Mathlib/Data/Sum/Basic.lean

```diff
@@ -464,6 +466,70 @@ theorem liftRel_swap_iff : LiftRel s r x.swap y.swap ↔ LiftRel r s x y :=
     exact h.swap, LiftRel.swap⟩
 #align sum.lift_rel_swap_iff Sum.liftRel_swap_iff
 
+theorem isLeft_eq_of_liftRel (h : LiftRel r s x y) : x.isLeft = y.isLeft := by
+  cases h <;> rfl
+
+theorem isRight_eq_of_liftRel (h : LiftRel r s x y) : x.isRight = y.isRight := by
+  rw [← Sum.not_isLeft, isLeft_eq_of_liftRel h, Sum.not_isLeft]
+
+theorem isLeft_eq_of_liftRel_inl_fst (h : LiftRel r s (inl a) y) : y.isLeft :=
+(isLeft_eq_of_liftRel h).symm
```

Comment:
Here and below:
```suggestion
  (isLeft_eq_of_liftRel h).symm
```

Reply:
Huh. I was sure I had recieved advice that said when I was giving a direct term (rather than a tactic proof) I shouldn't indent.